.babelrc

```diff
@@ -0,0 +1,13 @@
+{
+  "presets": [
+    ["env", {
+      "targets": {
+        "node": "4.0"
+      }
+    }]
+  ],
+  "plugins": [
+    "add-module-exports",
+    "transform-es2015-modules-commonjs"
```

Comment:
You are right. There was a problem with my NPM and dependencies and it wasn't working for me. Fixed. 👍